paddle/fluid/operators/distributed/grpc/grpc_client.cc

```diff
@@ -127,59 +127,74 @@ VarHandlePtr GRPCClient::AsyncGetVar(const std::string& ep,
                                      const platform::DeviceContext& ctx,
                                      const framework::Scope& scope,
                                      const std::string& var_name,
+                                     const std::string& out_varname,
                                      int64_t time_out) {
-  return _AsyncGetVar(ep, ctx, scope, var_name,
+  return _AsyncGetVar(ep, ctx, scope, "GetRPC", var_name, out_varname,
```

Comment:
use some const value to mark these rpc methed names? 
```cpp
constexpr char kGetRpc[] = "GetRPC";
constexpr char kGetNoBarrierRPC[] = "GetNoBarrierRPC";
```

Reply:
already have
https://github.com/PaddlePaddle/Paddle/pull/14854/files#diff-a2b16d4dc3fbf3cb87f7c4bffe9a8f89R45